frontend/src/components/LandingPage/Team/Team.js

```diff
@@ -0,0 +1,78 @@
+/*
+This component will have a title and description of the team members.
+*/
+
+import React from 'react';
+import { Card, Button, CardTitle, CardText, CardSubtitle, CardDeck} from 'reactstrap';
+import './Team.css'
+
+document.body.style.backgroundColor="#e3c3b2";
```

Comment:
Lets move all styling to the stylesheet. One way to change the bground color for the component is to add the color to the container div on line 13.